### PR TITLE
Update normalise.jl

### PR DESCRIPTION
**Pull request by namanbiyani**
#503 Provided aliases for unicode fields in batchnorm and dropout

### CODE REVIEW

File: src/layers/normalise.jl [hunk 1]
```diff
@@ -95,25 +95,25 @@ m = Chain(
...
+  BatchNorm(activation_function, param(initβ(chs)), param(initγ(chs)),
+            zeros(chs), ones(chs), epsilon, momentum, true)
 
 function (BN::BatchNorm)(x)
-  size(x, ndims(x)-1) == length(BN.β) ||
-    error("BatchNorm expected $(length(BN.β)) channels, got $(size(x, ndims(x)-1))")
-  γ, β = BN.γ, BN.β
+  size(x, ndims(x)-1) == length(BN.bias) ||
+    error("BatchNorm expected $(length(BN.bias)) channels, got $(size(x, ndims(x)-1))")
+  scale, bias = BN.scale, BN.bias
```
Comment (avik-pal) [inline src/layers/normalise.jl]: IMO it's cleaner to use unicode characters internally.

File: src/cuda/cudnn.jl [hunk 2]
```diff
@@ -194,7 +194,7 @@ end
 # Flux Interface
 
 (BN::Flux.BatchNorm)(x::Union{CuParam{T,2},CuParam{T,4},CuParam{T,5}}, cache = nothing) where T<:Union{Float32, Float64} =
-  batchnorm(BN.γ, BN.β, x, BN.μ, BN.σ², BN.momentum; cache = cache, alpha = 1, beta = 0, eps = BN.ϵ, training = BN.active)
+  batchnorm(BN.scale, BN.bias, x, BN.moving_mean, BN.moving_std, BN.momentum; cache = cache, alpha = 1, beta = 0, eps = BN.ϵ, training = BN.active)
```
Comment (CarloLucibello) [inline src/cuda/cudnn.jl]: `moving_std` should really be `moving_var`

**Comment by namanbiyani** [comment]
Please tell why the checks are getting failed . I am unable to understand this

File: test/layers/normalisation.jl [hunk 3]
```diff
@@ -78,7 +78,7 @@ end
     @test !m.active
 
     y = m(x).data
-    @test isapprox(y, data(sigmoid.((x .- m.μ) ./ sqrt.(m.σ² .+ m.ϵ))), atol = 1.0e-7)
+    @test isapprox(y, data(sigmoid.((x .- m.running_mean) ./ sqrt.(m.moving_var .+ m.eps))), atol = 1.0e-7)
```
Comment (avik-pal) [inline test/layers/normalisation.jl]: Should be 
```suggestion
    @test isapprox(y, data(sigmoid.((x .- m.running_mean) ./ sqrt.(m.running_var .+ m.eps))), atol = 1.0e-7)
```

**Comment by namanbiyani** [comment]
Please review @MikeInnes

**Comment by dhpollack** [comment]
@namanbiyani  Won't this cause problems with both #633 and #634 which both seem ready to go?  I actually agree the names should be unicode, but approving this and before those will be a bit of a headache.

**Comment by MikeInnes** [comment]
Note that as discussed in #503, we'll need to provide deprecation warnings for anything we change here.

**Comment by namanbiyani** [comment]
What should I do then @MikeInnes ?

**Comment by MikeInnes** [comment]
You can provide a deprecation by overloading `getproperty`, so that if someone tries to use the old names it'll work but with a warning to use the new name.
